metrics/local_test.go

```diff
@@ -9,14 +9,12 @@ import (
 )
 
 func TestLocalMetrics(t *testing.T) {
-	b := NewLocalBackend(0) // default interval
-	defer b.Stop()
-
 	tags := map[string]string{
 		"x": "y",
 	}
 
-	f := NewLocalFactory(b)
+	f := NewLocalFactory(0)
```

Comment:
should we simplify this?  Right now passing 0 as interval changes it to 1min. I would remove this default and instead avoid starting the go-routine when it's 0. In this case calling Stop from tests is not required (but should still be allowed).